server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectionAlgorithm.java

```diff
@@ -183,7 +183,11 @@ private void removeCandidatesInUse(GarbageCollectionEnvironment gce,
         GcCandidate gcTemp = candidateMap.remove(relativePath);
         if (gcTemp != null) {
           log.debug("File Candidate was still in use: {}", relativePath);
-          inUseCandidates.add(gcTemp);
+          // Intentionally not adding scan candidates to inUseCandidates as they are only added
+          // once.
```

Comment:
Presumably, all the code we write is intentional, so I think that part of this comment isn't necessary. But more importantly, I think this comment isn't quite clear because it's not clear at this point in the code what the data structure "inUseCandidates" is expected to be used for. A variable name based on a full description would be something like: "candidatesForDeletionStillInUseAndCanBeDeletedBecauseANewCandidateWillBeCreatedLater".

Obviously, that's horrible. But, perhaps if the name were something like "candidateEntriesToBeDeleted", then the comment could be something like "Prevent deleting candidate entries that are still in-use only by scans, because they won't be recreated when the scan is finished." I think the key part here is trying to distinguish between deletion of candidate entries and the candidate files themselves.

Reply:
Changed the variable name from `inUseCandidates` to `candidateEntriesToBeDeleted` to help with readability. 
Also updated the comment wording as well.